lib/src/react_client/synthetic_event_wrappers.dart

```diff
@@ -148,7 +148,7 @@ class SyntheticEvent {
   ///
   /// > See: [persist]
   @Deprecated('The modern event system does not use pooling. This always returns true.')
-  external bool isPersistent();
+  bool get isPersistent => true;
```

Comment:
Hmmm related to this change, it looks like this needs to be external: https://wf-skynet-hrd.appspot.com/apps/test/smithy/2552158/1/graph_ui/unit-tests#summary

Trying it as an extension instead :loading:

Reply:
Yep, the extension worked: https://wf-skynet-hrd.appspot.com/apps/test/smithy/2552158/3